app/src/main/java/org/p2p/wallet/sell/interactor/SellInteractor.kt

```diff
@@ -27,9 +29,21 @@ class SellInteractor(
     private val homeLocalRepository: HomeLocalRepository,
     private val tokenKeyProvider: TokenKeyProvider,
     private val userInteractor: UserInteractor,
-    private val hiddenSellTransactionsStorage: HiddenSellTransactionsStorageContract
+    private val hiddenSellTransactionsStorage: HiddenSellTransactionsStorageContract,
+    private val sharedPreferences: SharedPreferences,
 ) {
 
+    companion object {
+        const val SHOW_SELL_INFORM_DIALOG_KEY = "SHOW_SELL_INFORM_DIALOG_KEY"
+    }
+
+    fun isNeedShowInformDialog(): Boolean =
```

Comment:
i would rather put it in the presenter, it is not domain or reusable logic

Reply:
`setShouldShowInformDialog` and `shouldShowInformDialog` is pair